paddle/fluid/framework/paddle2cinn/transform_desc.h

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include "paddle/fluid/framework/block_desc.h"
+#include "paddle/fluid/framework/op_desc.h"
+#include "paddle/fluid/framework/program_desc.h"
+#include "paddle/fluid/framework/var_desc.h"
+
+#include "cinn/frontend/paddle/cpp/block_desc.h"
+#include "cinn/frontend/paddle/cpp/desc_api.h"
+#include "cinn/frontend/paddle/cpp/op_desc.h"
+#include "cinn/frontend/paddle/cpp/program_desc.h"
+#include "cinn/frontend/paddle/cpp/var_desc.h"
+
+namespace paddle {
+namespace framework {
+namespace paddle2cinn {
+
+namespace cpp = ::cinn::frontend::paddle::cpp;
```

Comment:
A small C++ knowledge and habit: try to avoid `using namespace` and `namespace = ` in header file because it can break the encapsulation of namespace.

Let's say, if other file `#include<transform_desc.h>`, then they can also call `cpp:XxDesc` to access the variable inside the namespace.

If you write `using namespace ::cinn::frontend::paddle::cpp`, that's worse, other file can just call `XxDesc`

So above breaks namespace encapsulation. However, another C++ knowledge is that it is OK to write `using namespace` and `namespace = ` in source file.

Reply:
Thanks, you are right! I'll fix it in next commit.